types/web-element.d.ts

```diff
@@ -360,10 +360,9 @@ export type ScopedElementRect = {
   readonly height: number;
 };
 
-export type DragAndDropDestination = {
-  readonly xOffset: number;
-  readonly yOffset: number;
-};
+export type DragAndDropDestination = 
+|{readonly x: number; readonly y: number;}
+|WebElement
```

Comment:
minor lint fixes (proper spaces), otherwise LGTM!
```
export type DragAndDropDestination = 
  | {readonly x: number; readonly y: number;}
  | WebElement
```

@garg3133